framework/auth/__init__.py

```diff
@@ -35,11 +39,8 @@ def authenticate(user, access_token, response):
         'auth_user_access_token': access_token,
     })
     print_cas_log(f'Finalizing authentication - data updated: user=[{user._id}]', LogLevel.INFO)
-    user.update_date_last_login()
-    user.clean_email_verifications()
-    user.update_affiliated_institutions_by_email_domain()
-    user.save()
-    print_cas_log(f'Finalizing authentication - user updated: user=[{user._id}]', LogLevel.INFO)
+    enqueue_task(update_user_from_login.s(user._id, timezone.now(), user_updates))
```

Comment:
Ah, I see.  We use `enqueue_task()` instead of `.delay()` here due to task being called during request.